### PR TITLE
fix qwen3-omni audio length < 30s

### PR DESCRIPTION
**Pull request by jiapingW**
## Motivation

The qwen3 Omni series models, due to their use of the WhisperFeatureExtractor class, limit the chunk size to 30 seconds, causing truncation regardless of the input length. It would be desirable to support Omni models in processing complete audio. Fix the bug [here](https://github.com/sgl-project/sglang/issues/12610).
## Checklist

- [✅] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).

### CODE REVIEW

**Review by mickqian — APPROVED** [review]
LGTM for this one, but eventually we should adopt user-overriding processor kwargs

**Comment by jiapingW** [comment]
Yes. I think the relevant parameters, such as those shown below, should be encapsulated in sglang by inheriting from transformers; otherwise, it's difficult to synchronize small updates to transformers to sglang.
```python
        "text_kwargs": {
            "padding": False,
            "padding_side": "left",
        },
        "videos_kwargs": {
            "seconds_per_chunk": 2.0,
            "position_id_per_seconds": 13.0,
            "use_audio_in_video": False,
            "size": {
                "shortest_edge": 128 * 32 * 32,
                "longest_edge": 768 * 32 * 32,
            },
        },
        "audio_kwargs": {
            "sampling_rate": 16000,
            "padding": True,
            "truncation": False,
            "return_attention_mask": True,
        },
```
> LGTM for this one, but eventually we should adopt user-overriding processor kwargs
